includes/admin_import.php

```diff
@@ -4,743 +4,1212 @@
+									<p>To sign up for Mapbox <a href="https://www.mapbox.com/signup/" target="_blank">go here</a>. You will only need
+										a valid email address, no credit card required. Copy your access token and paste it below:</p>
+									<form class="columns" method="post" action="<?php echo $_SERVER['REQUEST_URI'] ?>" style="margin:15px -5px 22px 0;">
+										<?php wp_nonce_field($tsml_nonce, 'tsml_nonce', false) ?>
+										<div class="input">
+											<input type="text" name="tsml_add_mapbox_key" placeholder="Enter Mapbox access token here">
+										</div>
+										<div class="btn">
+											<input type="submit" class="button" value="<?php _e('Add', '12-step-meeting-list') ?>">
+										</div>
+									</form>
+
+									<p>*Please note: Only Mapbox is supported by our <b>TSML UI</b> user interface!  Google is not an option.
+
+									<p>For our legacy user interface (<b>Legacy UI</b>), you can alternatively choose to use Google. Their interface is slightly more complex because they offer more
+										services. <a href="https://developers.google.com/maps/documentation/javascript/get-api-key" target="_blank">Go here</a>
+										to get a key from Google. The process should only take a few minutes, although you will have to enter a	credit card.
+										<a href="https://theeventscalendar.com/knowledgebase/setting-up-your-google-maps-api-key/" target="_blank">Here
+											are some instructions</a>.
+									</p>
+
+									<p>Be sure to:<br>
+										<span class="dashicons dashicons-yes"></span> Enable the Google Maps Javascript API<br>
+										<span class="dashicons dashicons-yes"></span> Secure your credentials by adding your website URL to the list
+										of allowed referrers
+									</p>
+
+									<p>Once you're done, paste your new key below.</p>
+
+									<form class="columns" method="post" action="<?php echo $_SERVER['REQUEST_URI'] ?>" style="margin:15px -5px 22px 0;">
+										<?php wp_nonce_field($tsml_nonce, 'tsml_nonce', false) ?>
+										<div class="input">
+											<input type="text" name="tsml_add_google_maps_key" placeholder="Enter Google API key here">
+										</div>
+										<div class="btn">
+											<input type="submit" class="button" value="<?php _e('Add', '12-step-meeting-list') ?>">
+										</div>
+									</form>
+								</div>
+							</div>
+						<?php } ?>
+					</div>
+				</div>
+			</div>
+
+			<nav class="nav-tab-wrapper" >
+				<a href="?post_type=tsml_meeting&page=import" class="nav-tab <?php if($tab===null):?>nav-tab-active<?php endif; ?>">Import</a>
+				<a href="?post_type=tsml_meeting&page=import&tab=settings" class="nav-tab <?php if($tab==='settings'):?>nav-tab-active<?php endif; ?>">Settings</a>
+				<a href="?post_type=tsml_meeting&page=import&tab=example" class="nav-tab <?php if($tab==='example'):?>nav-tab-active<?php endif; ?>">Spreadsheet Example & Spec</a>
+			</nav>
+
+			<div class="tab-content">
+			<?php switch($tab) :
```

Comment:
perhaps these tabs could be in includes? might make this file a bit easier to read. eg `admin_import_[tab].php`

Reply:
Still **TODO**